letsencrypt-auto

```diff
@@ -41,6 +41,9 @@ then
   elif [ -f /etc/arch-release ] ; then
     echo "Bootstrapping dependencies for Archlinux..."
     $SUDO $BOOTSTRAP/archlinux.sh
+  elif [ -f /etc/manjaro-release ] ; then
+    echo "Bootstrapping dependencies for Manjaro linux..."
+    $SUDO $BOOTSTRAP/archlinux.sh
```

Comment:
So... you created `manjaro.sh`, but you don't use it?

BTW I'd suggest that you follow the same approach as we have for Debian-derivatives: `debian.sh` and `ubuntu.sh` are symlinks to `_deb_common.sh`. In case something diverges we'll separate the files